### PR TITLE
[Bugfix] [seatunnel-core-spark] plugin jars can't be added correctly in SparkStarter

### LINKED ISSUE #1818: Seatunnel can't load the user defined plugin jars
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

Jars like oracle、mysql driver jars puted in plugins/[pluginName]/lib ， would not be loaded where seatunnel started

### SeaTunnel Version

2.1.1

### SeaTunnel Config

```conf
nothing
```

### Running Command

```shell
nothing
```

### Error Exception

```log
The exceptions like db driver class not find
```

### Flink or Spark Version

spark2.4

### Java or Scala Version

java8

### Screenshots

image
image
image

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by chenhu**
## Purpose of this pull request

close #1818

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-core/seatunnel-core-spark/src/main/java/org/apache/seatunnel/core/spark/SparkStarter.java [hunk 1]
```diff
@@ -191,11 +191,15 @@ private List<Path> getPluginsJarDependencies() throws IOException {
         if (!Files.exists(pluginRootDir) || !Files.isDirectory(pluginRootDir)) {
             return Collections.emptyList();
         }
+        //my test code
+        Path pluginJarPath = Paths.get("/Users/chenhu/projects/org/apache/seatunnel/seatunnel-common/plugins/oracle/lib/ojdbc8.jar");
+
```
Comment (wuchunfu) [inline seatunnel-core/seatunnel-core-spark/src/main/java/org/apache/seatunnel/core/spark/SparkStarter.java]: This path seems to be problematic, not general enough, please modify it, thanks

**Comment by wuchunfu** [comment]
@chenhu Please modify the pr title, thanks.

File: seatunnel-core/seatunnel-core-spark/src/main/java/org/apache/seatunnel/core/spark/SparkStarter.java (hunk 1 above)
Comment (chenhu) [reply seatunnel-core/seatunnel-core-spark/src/main/java/org/apache/seatunnel/core/spark/SparkStarter.java]: o, my mistake

**Comment by chenhu** [comment]
> 

the tile changed

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Pull request merged**